src/components/v5/common/CompletedAction/partials/MultiSigMeatballMenu/MultiSigMeatballMenu.tsx

```diff
@@ -73,6 +75,15 @@ const MultiSigMeatballMenu: FC<MultiSigMeatballMenuProps> = ({
     setShowRejectMultiSigStep(true);
   };
 
+  const canCancelMultiSig =
+    !isRejected &&
+    (isOwner ||
+      Date.now() - new Date(multiSigData.createdAt).getTime() >
+        MULTISIG_EXPIRY_PERIOD);
+
+  const canRejectMultiSig =
+    !canCancelMultiSig && !isRejected && !isOwner && !showRejectMultiSigStep;
```

Comment:
I wonder if this variable should be renamed to `canRejectImmediately` / `canCancelWithoutVote` or something to avoid confusion between cancel and reject?